doc/src/pair_lj_relres.rst

```diff
@@ -0,0 +1,282 @@
+        \sum_{m=0}^{4} \gamma_{sm}\left(r-r_{si}\right)^m-\Gamma_{so} ,   & \quad\mathrm{if}\quad  r_{si}\leq r< r_{so}, \\
+        4 \epsilon^{\scriptscriptstyle CG} \left[ \left(\frac{\sigma^{CG}}{r}\right)^{12} -     \left(\frac{\sigma^{CG}}{r}\right)^6 \right]-\Gamma_c, &  \quad\mathrm{if}\quad  r_{so}\leq r<r_{ci}, \\
+        \sum_{m=0}^{4} \gamma_{cm}\left(r-r_{ci}\right)^m -\Gamma_c, & \quad\mathrm{if}\quad  r_{ci}\leq r< r_{co}, \\
+        0, & \quad\mathrm{if}\quad  r\geq r_{co}.\end{array}\right.
+
+The FG parameters of the LJ potential (:math:`\epsilon^{FG}` and
+:math:`\sigma^{FG}`) are applied up to the inner switching distance,
+:math:`r_{si}`, while the CG parameters of the LJ potential
+(:math:`\epsilon^{CG}` and :math:`\sigma^{CG}`) are applied beyond the
+outer switching distance, :math:`r_{so}`. Between :math:`r_{si}` and
+:math:`r_{so}` a polynomial smoothing function is applied so that the
+force and its derivative are continuous between the FG and CG
+potentials.  An analogous smoothing function is applied between the
+inner and outer cutoff distances (:math:`r_{ci}` and :math:`r_{co}`).
+The offsets :math:`\Gamma_{si}`, :math:`\Gamma_{so}` and
+:math:`\Gamma_{c}` ensure the continuity of the energy over the entire
+domain.  The corresponding polynomial coefficients :math:`\gamma_{sm}`
+and :math:`\gamma_{cm}`, as well as the offsets are automatically
+computed by LAMMPS.
+
+.. note::
+
+   Energy and force resulting from this methodology can be plotted via the
+   :doc:`pair_write <pair_write>` command.
+
+The following coefficients must be defined for each pair of atom types
+via the :doc:`pair_coeff <pair_coeff>` command as in the examples above,
+or in the data file or restart files read by the :doc:`read_data
+<read_data>` or :doc:`read_restart <read_restart>` commands, or by
+mixing as will be described below:
+
+* :math:`\epsilon^{FG}` (energy units)
+* :math:`\sigma^{FG}` (distance units)
+* :math:`\epsilon^{CG}` (energy units)
+* :math:`\sigma^{CG}` (distance units)
+
+Additional parameters can be defined to specify different
+:math:`r_{si}`, :math:`r_{so}`, :math:`r_{ci}`, :math:`r_{co}` for
+a particular set of atom types:
+
+* :math:`r_{si}` (distance units)
+* :math:`r_{so}` (distance units)
+* :math:`r_{ci}` (distance units)
+* :math:`r_{co}` (distance units)
+
+These parameters are optional, and they are used to override the global
+cutoffs as defined in the pair_style command.  If not specified, the
+global values for :math:`r_{si}`, :math:`r_{so}`, :math:`r_{ci}`, and
+:math:`r_{co}` are used.  If this override option is employed, all four
+arguments must be specified.
+
+----------
+
+Here are some guidelines for using the pair_style *lj/relres* command.
+
+At the most basic level in the RelRes framework, groups of atoms must be
```

Comment:
Another point to clarify to define clusters.  If you have a simple all-atom polymer chain with a C backbone and H atoms attached, and you want each C atom and its 2 H atoms to be a "cluster".  Then there are also C-C bonds, but presumably those don't count as augmenting each cluster?  If so, then that should also be explained.

Reply:
One more Q: does this pair style have some requirement for how the newton pair or bond flags are set?  Or for the weighting coeffs of the special_bond command?  Is so, that should be mentioned in this doc page.  I'm wondering how generally you identify clusters within the code.  I.e. whether it works for any settings of those parameters.  Since both of them affect what I,J pairs are in the neighbor list and what bonds each atom knows about.

Reply:
Thank you for your constructive comments. We made the appropriate changes. If there are any other issues, please, let us know.

Reply:
Doc changes look good - thanks.  But I didn't see any mention or answer to this Q:
One more Q: does this pair style have some requirement for how the newton pair or bond flags are set? Or for the weighting coeffs of the special_bond command? Is so, that should be mentioned in this doc page. I'm wondering how generally you identify clusters within the code. I.e. whether it works for any settings of those parameters. Since both of them affect what I,J pairs are in the neighbor list and what bonds each atom knows about.
Have you run your new pair style in parallel with newton on/off for both pair and bond (4 permutations).  Ditto with different special_bond settings.  To see if the answers change or something breaks?

Reply:
Steve, we thought that this paragraph gives the answer to your question:
"In general, RelRes focuses on the speedup of pairwise interactions between all LJ sites. Importantly, it does not at all effect any other settings, flags, etc. that would be used in a molecular simulation with the conventional LJ potential. In particular, all intramolecular topology with its energetics (i.e., bonds, angles, etc.) remains unaltered."
Specifically, following your request, we ran a quick test with all permutations of the newton flags, as well as while varying the special_bond command: Everything is the same, and nothing breaks.  If you think we need to add any phrases in the documentation, please let us know.

Reply:
ok - great - thanks for doing the tests.  In this paragraph then I would just change the wording from:
>> it does not at all effect any other settings, flags, etc. that would be used with ...
to
>> it works with other settings and flags (e.g. newton flags, special_bonds settings) that can be used for ...
And mark "newton" and "special_bonds" with a link to those commands.

Reply:
Done! Have a good weekend!